src/components/issues/new-issue/NewIssue.js

```diff
@@ -72,7 +71,4 @@ const Wrapper = styled.div`
   justify-content: space-between;
 `;
 
-const mapStateToProps = (state: StateType) => ({
-  userInfo: state.user.userInfo
-});
-export default withRouter(connect(mapStateToProps, { addNewIssue })(NewIssue));
+export default withRouter(connect(null, { addNewIssue })(NewIssue));
```

Comment:
no need to change now but when you have multiple HoC you can use compose function from redux to write it in a way that's more readable. 

e.g:
```js
export default compose(
 withRouter,
 connect(null, { addNewIssue })
)(NewIssue)
```